inst/scratch.R

```diff
@@ -0,0 +1,27 @@
+epidist_family.epidist_latent_individual <- function(data,
+                                                     family = "lognormal",
+                                                     ...) {
+  epidist_validate(data)
+  # allows use of stats::family and strings
+  family <- brms:::validate_family(family)
```

Comment:
Does this have a class and we can dispatch on it?

Reply:
Presumably it must have some kind of identifier about what family it is

Reply:
Hopefully no downstream effects of classing the `family`

Reply:
Seems like it gets rid of the class when you make the custom family out of it so fine